RATapi/examples/non_polarised/DSPC_standard_layers.ipynb

```diff
@@ -0,0 +1,536 @@
+    "          hydration=\"Bilayer Tails Hydration\", hydrate_with=\"bulk out\")\n",
+    "]\n",
+    "\n",
+    "problem.layers.extend(layers)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "356964f9-83a6-4a92-8092-4d250b68ac16",
+   "metadata": {},
+   "source": [
+    "Now deal with the experimental parameters. We need a bulk in of Silicon, and two Bulk outs - D2O and SMW."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "id": "b197f3ea-c6ef-4831-9500-9ff0cb5011f3",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "del problem.bulk_in[0]\n",
+    "problem.bulk_in.append(name=\"Silicon\", min=2.0e-06, value=2.073e-06, max=2.1e-06, fit=False)\n",
+    "\n",
+    "del problem.bulk_out[0]\n",
+    "problem.bulk_out.append(name=\"D2O\", min=5.5e-06, value=5.98e-06, max=6.4e-06, fit=True)\n",
+    "problem.bulk_out.append(name=\"SMW\", min=1.0e-06, value=2.21e-06, max=4.99e-06, fit=True)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "420b57a9-4fc7-49d5-acaa-68570f1876d2",
+   "metadata": {},
+   "source": [
+    "Likewise the scalefactors and backgrounds:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "id": "92a26ca2-b1ee-41c8-89ce-8b72c0892438",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "del problem.scalefactors[0]\n",
+    "problem.scalefactors.append(name=\"Scalefactor 1\", min=0.05, value=0.10, max=0.2, fit=False)\n",
+    "problem.scalefactors.append(name=\"Scalefactor 2\", min=0.05, value=0.15, max=0.2, fit=False)\n",
+    "\n",
+    "# Now deal with the backgrounds\n",
+    "del problem.backgrounds[0]\n",
+    "del problem.background_parameters[0]\n",
+    "problem.background_parameters.append(name=\"Background parameter D2O\", min=5.0e-10, value=2.23e-06, max=7.0e-06, fit=True)\n",
+    "problem.background_parameters.append(name=\"Background parameter SMW\", min=1.0e-10, value=3.38e-06, max=4.99e-06, fit=True)\n",
+    "\n",
+    "problem.backgrounds.append(name=\"D2O Background\", type=\"constant\", value_1=\"Background parameter D2O\")\n",
+    "problem.backgrounds.append(name=\"SMW Background\", type=\"constant\", value_1=\"Background parameter SMW\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "a1b04d8b-8cc8-4e35-9e06-a6be3de90c53",
+   "metadata": {},
+   "source": [
+    "Now load in and add the data:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "id": "73681532-2688-4bfd-8153-1ab763f5687a",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "data_path = os.path.join(pathlib.Path.cwd().parents[0].resolve(), \"data\")\n",
+    "\n",
+    "d2o_dat = np.loadtxt(os.path.join(data_path, \"DSPC_D2O.dat\"), delimiter=\",\")\n",
+    "problem.data.append(name=\"dspc_bil_D2O\", data=d2o_dat)\n",
+    "\n",
+    "smw_dat = np.loadtxt(os.path.join(data_path, \"DSPC_SMW.dat\"), delimiter=\",\")\n",
+    "problem.data.append(name=\"dspc_bil_smw\", data=smw_dat)"
```

Comment:
`os.path` and `pathlib` are designed to be pretty much orthogonal: you can use `pathlib` entirely here rather than this mix and it's a lot clearer to the reader what you're actually doing:
```python
data_path =  "../data"
d2o_dat = data_path / "DSPC_D2O.dat"
smw_dat = data_path / "DSPC_SMW.dat"
```
same for the other tutorials when we add datafiles